spec/models/update_user_spec.rb

```diff
@@ -1,37 +1,44 @@
 require 'rails_helper'
 
 RSpec.describe UpdateUser do
-  let(:updater) { UpdateUser.new(params, current_user) }
-  let(:current_user) { User.create(duns_number: '12341', sam_account: true) }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:updater) { UpdateUser.new(params, user) }
 
+  let(:user_id) { user.id }
+  let(:user_name) { Faker::Name.name }
+  let(:user_email) { Faker::Internet.email }
+  let(:user_duns) { user.duns_number }
+  let(:user_credit_card_url) { 'https://random-example.com/pay' }
```

Comment:
I generally like to avoid `let` statements - leads to a mystery guest problem where you're referencing these vars in a long file. More here: https://robots.thoughtbot.com/lets-not